lectures/11.rendering_2.md

```diff
@@ -320,9 +321,10 @@ GPU Stalls
 GPU – Hiding STALLS
 
 * Idea, saturate the gpu with work.
-* Issue data request for item N, block
-* Issue data request for item N + 1, block
-* Issue data request for item N + 2, block
+
+* Issue data request for item N, block <!-- .element class="fragment" data-fragment-index="0" -->
+* Issue data request for item N + 1, block <!-- .element class="fragment" data-fragment-index="1" -->
+* Issue data request for item N + 2, block <!-- .element class="fragment" data-fragment-index="2" -->
 
 * Data comes for item N, execute
 * Data comes for item N + 1, execute

```

Comment:
you animated the "Issue data request" but not the "data comes part"; I guess you meant to do both